Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: build push run shell
```

Comment:
The `Makefile` won't be executed by the pipeline: it should be deleted

Reply:
Indeed, I removed the Makefile and update the readme to reference the one used across the jenkins-infra project